src/scripts/modules/media/body/body.coffee

```diff
@@ -469,7 +469,12 @@ define (require) ->
     toggleSolution: (e) ->
       $solution = $(e.currentTarget).closest('.solution, [data-type="solution"]')
       $solution.toggleClass('ui-solution-visible')
-
+      if $solution.hasClass('ui-solution-visible')
```

Comment:
Can you remove this line and update the CSS to use `[aria-expanded="true"]` instead of `.ui-solution-visible`?